jpo-geojsonconverter/src/test/java/us/dot/its/jpo/geojsonconverter/utils/ProjectionUtilsTest.java

```diff
@@ -0,0 +1,96 @@
+package us.dot.its.jpo.geojsonconverter.utils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
```

Comment:
issue (blocking): surefire is configured to use Junit 4 for this project, so this test doesn't get run with the jupiter Test annotation.
suggestion: use junit 4 annotation
```suggestion
import org.junit.Test;
```